tests/test_sinks.py

```diff
@@ -23,8 +24,7 @@ def __call__(self, req, resp, **kwargs):
 
 
 class SinkAsync(Sink):
-    async def __call__(self, req, resp, **kwargs):
```

Comment:
Reimplementing `__call__` is intended here (note the change from sync to `async`).